keyboards/rocketboard_16/keycode_lookup.h

```diff
@@ -0,0 +1,37 @@
+/* Copyright 2021 Seth Bonner <fl3tching101@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "quantum.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef struct
+{
+    char key_string[17];
+    uint16_t keycode;
+} lookup_table_t;
+
+char* translate_keycode_to_string(uint16_t code);
+
+extern lookup_table_t lookup_table[366];
+
+#ifdef __cplusplus
+}
+#endif
```

Comment:
```suggestion
typedef struct
{
    char key_string[17];
    uint16_t keycode;
} lookup_table_t;

char* translate_keycode_to_string(uint16_t code);

extern lookup_table_t lookup_table[366];
```

Reply:
Fixed this in [e5590bc](https://github.com/qmk/qmk_firmware/pull/12537/commits/e5590bcec542a8a82b2c2bb5f34f332b59885d87)